lib/sycamore/sycamore/query/schema.py

```diff
@@ -21,15 +21,16 @@ def get_schema(self) -> dict[str, str]:
 
         for k, v in schema[self._index]["mappings"].items():
             if k.startswith("properties.entity") and ".keyword" not in k:
```

Comment:
should we also add a check for if k has a "." in it? this could reduce the # of exceptions we have to catch